src/components/content/Content.module.sass

```
@@ -2,24 +2,32 @@
     height: 100%
     display: flex
     overflow: hidden
+    position: relative
 
     .main-content
         flex-grow: 1
         display: grid
         grid-template-rows: 1fr auto
-        overflow: auto
+        overflow-y: auto
 
         main
             padding-block: 2rem
             max-width: calc( 100% - 16rem )
```

Because the vertical scrollbar is in the middle of the page (in the end of the main element), I recommand to expand the component width to 100% and add padding instead.
```suggestion
            padding-inline-end: 16rem
```